### PR TITLE
Launch multiple fc analysis from ngi_pipeline_start.py

### PR DESCRIPTION
**Pull request by mariogiov**

### CODE REVIEW

File: scripts/ngi_pipeline_start.py [hunk 1]
```diff
@@ -133,17 +137,18 @@ def __init__(self, *args, **kwargs):
...
-                                                args.restrict_to_projects,
-                                                args.restrict_to_samples,
-                                                args.restart_failed_jobs,
-                                                args.restart_finished_jobs,
-                                                args.restart_running_jobs,
-                                                quiet=args.quiet,
-                                                manual=True)
+    if 'analyze_fc_dirs' in args:
+        LOG.info('Starting flowcell analysis of flowcell {} '
+                 '{}'.format(inflector.plural("directory", len(args.analyze_fc_dirs)),
```
Comment (guillermo-carrasco) [inline scripts/ngi_pipeline_start.py]: hmm are you using this librarie just not to write `directories`? Am I missing something? :sweat_smile:

Comment (mariogiov) [inline scripts/ngi_pipeline_start.py]: The output is more terse! It's Pythonic! Right? Right?

Comment (mariogiov) [inline scripts/ngi_pipeline_start.py]: Imagine all the use cases -- it's every noun in the English language! What a module!

Comment (guillermo-carrasco) [inline scripts/ngi_pipeline_start.py]: sure... :sweat_smile:

**Pull request merged**